docs/modules/deploy/pages/deploying-on-aws.adoc

```diff
@@ -221,19 +224,22 @@ config.getNetworkConfig().getInterfaces().setEnabled(true).addInterface("10.0.*.
 --
 ====
 
-The following optional properties can be configured:
+The following optional properties can be configured[[ecs_properties]]:
 
 * `access-key`, `secret-key`: access and secret keys of AWS your account; if not set, IAM Task Role is used
 * `region`: region where Hazelcast members are running; default is the current region
 * `cluster`: ECS cluster short name or ARN; default is the current cluster
 * `family`: filter to look only for ECS tasks with the given family name; mutually exclusive with `service-name`
 * `service-name`: filter to look only for ECS tasks from the given service; mutually exclusive with `family`
+* `tag-key`, `tag-value`: filter to look only for ECS Tasks with the given `tag-key`/`tag-value`; multi values supported if comma-separated (e.g. `KeyA,KeyB`); comma-separated values behaves as AND conditions
```

Comment:
`tag-key`, `tag-value`: filter to look only for ECS Tasks with the given `tag-key`/`tag-value`; comma-separated multiple values are supported, e.g., `KeyA,KeyB`; comma-separated values behave as AND conditions